plugins/repomanage.py

```diff
@@ -84,9 +84,10 @@ def run(self):
                 module_packages = self.base._moduleContainer.getModulePackages()
 
                 for module_package in module_packages:
-                    all_modular_artifacts.update(module_package.getArtifacts())
-                    module_dict.setdefault(module_package.getNameStream(), {}).setdefault(
-                        module_package.getVersionNum(), []).append(module_package)
+                    if module_package.getRepoID() == "repomanage_repo":
```

Comment:
Please can you add a comment that will clarify why you do it?
It is clear that only one available repository is loaded "repoman age_repo", but the workflow also loads Fail-safe data stored on the disk. Please can you point out it somehow?
Please can you also add an information that the issue was caused by loading of failsafe data?

Reply:
Added. :+1:

Reply:
Please can you define `"repomanage_repo"` as local variable? Like `REPOMANAGE_REPONAME`?

Reply:
Please can you also add to the commit header `
repomanage: Use modules only from repo they are handling ` `RhBug:` reference? Or is it not needed for generation of upstream changelog?

Reply:
Ok, it is not needed anymore but it is nice to see it in the header.

Reply:
Both issues resolved.